addons/project/static/src/views/project_form/project_form_view.scss

```diff
@@ -0,0 +1,11 @@
+.o_form_project_project {
+    .o_widget_web_ribbon:not(.o_invisible_modifier) + div div[name="last_update_status"] > div {
+        margin-right: 150px;
+        padding-left: 1rem;
+    }
+
+    div[name="last_update_status"] span.o_status {
+        width: 17px;
+        height: 17px;
+    }
+}
```

Comment:
So, here, to avoid duplicating code, you could:
1) Extend StateSelectionField ('kanban_state') for tasks:
export class Blablabla extends StateSelectionField {
    ...
}
ProjectStateSelectionField.additionalClasses = ['blablabla'];
2) Replace 'kanban_state' with 'blablabla' in the css files of the project module.
3) Extend Blabla for what you want to do in with the ProjectStateSelectionField widgets